cloud/azure/modules/app/variables.tf

```diff
@@ -76,17 +65,18 @@ variable "resource_group_name" {
 variable "postgres_admin_login" {
   type        = string
   description = "Postgres admin login"
+  default     = "psqladmin"
```

Comment:
Does this need to be a fixed value. The AWS side is generated during the initial setup process?

Reply:
I had it be fixed because I liked connecting to the DB via azure powershell and not having to look up the username every time. We can change it to be generated following the pattern of {adjective}-{noun} that we use right now for the app service instance but I'm not sure of the value, other than it follows the AWS pattern (and maybe that suffices). 

For now I would like to keep it as psqladmin because changing it would mean modifying this PR further and it's swelled to a large size already. I can add a note to discuss in our next team meeting as to whether the database username should be generated vs. fixed.

Reply:
yup, can change in a followup PR.